### PR TITLE
age Group

### PR DESCRIPTION
**Pull request by ilovemane**

### CODE REVIEW

**Comment by catalamarti** [comment]
argument to determine if `multiple` age groups are allowed or not

File: R/validate.R [hunk 1]
```diff
@@ -227,6 +227,127 @@ assertValidation <- function(validation, call = parent.frame()) {
...
+#' @return validate ageGroup
+#' @export
+#'
+validateAgeGroupArgument <- function(ageGroup,
+                                     multipleAgeGroup = TRUE,
+                                     overlap = FALSE,
+                                     call = parent.frame()){
+
+  assertList(ageGroup, null = TRUE, call = call)
+  assertLogical(multipleAgeGroup, length = 1)
```
Comment (catalamarti) [inline R/validate.R:246]: pass call argument

File: R/validate.R [hunk 2]
```diff
@@ -227,6 +227,127 @@ assertValidation <- function(validation, call = parent.frame()) {
...
+  assertLogical(overlap, length = 1)
```
Comment (catalamarti) [inline R/validate.R:247]: pass call argument

File: R/validate.R [hunk 3]
```diff
@@ -227,6 +227,127 @@ assertValidation <- function(validation, call = parent.frame()) {
...
+#' @param ageGroup age group in a list
+#' @param multipleAgeGroup allow mutliple age group
+#' @param overlap allow overlapping ageGroup
+#' @param call parent frame
+#'
+#' @return validate ageGroup
+#' @export
+#'
+validateAgeGroupArgument <- function(ageGroup,
+                                     multipleAgeGroup = TRUE,
```
Comment (catalamarti) [inline R/validate.R:241]: should default be false? to discuss @edward-burn

File: R/validate.R [hunk 4]
```diff
@@ -227,6 +227,127 @@ assertValidation <- function(validation, call = parent.frame()) {
...
+                                     overlap = FALSE,
+                                     call = parent.frame()){
```
Comment (catalamarti) [inline R/validate.R:243]: add argument null = TRUE/FALSE to assess if NULL is allowed or not

**Review by catalamarti — CHANGES_REQUESTED** [review]

File: R/validate.R (hunk 4 above)
Comment (ilovemane) [reply R/validate.R:245]: added @catalamarti

**Review by catalamarti — APPROVED** [review]

**Pull request merged**